### DISCUSSION TITLE
Change aria properties of QRating to be compliant with W3 WAI (Accessibility)

### DISCUSSION THREAD

**Discussion by pieterzeilstra** [quasarframework/quasar#13941] [Ideas / Proposals] [answered]
When navigating a QRating component with a screenreader (e.g. NVDA) it does not give good feedback (it only says it clickable). and the navigation uses tabs instead of the keyboard compared to the example from W3.
https://www.w3.org/WAI/ARIA/apg/example-index/radio/radio-rating.html

It would be nice if the aria properties were better so people can navigate quasar applications 😄 .

**Comment by rstoenescu** [comment, marked as answer]
New prop "icon-aria-label" (String or Array) will be available in Quasar v2.8.5 and v1.20.3
